src/xbps/updating.md

```diff
@@ -1,47 +1,37 @@
 # Updating
 
-Like any other system it is important to keep Void Linux up to date.
-
-In general Void should be updated with an XBPS invocation:
+Like any other system, it is important to keep Void up-to-date. Use
+[xbps-install(1)](https://man.voidlinux.org/xbps-install.1) to update:
 
 ```
 # xbps-install -Su
 ```
 
-> Note: XBPS must use a separate transaction to update itself. If your first
-> update includes the package `xbps`, you will need to run an additional update
-> for the rest of the system.
+XBPS must use a separate transaction to update itself. If your update includes
+the `xbps` package, you will need to run the above command a second time to
+apply the rest of the updates.
 
-## Restarting services
+## Restarting Services
 
-If you are installing Void in production or otherwise have long lived services,
-its important to note that XBPS does not restart services when they are updated.
-This task is left to the administrator so they can orchestrate maintenance
-windows, ensure reasonable backup capacity, and generally be present for service
-upgrades.
+XBPS does not restart services when they are updated. This task is left to the
+administrator, so they can orchestrate maintenance windows, ensure reasonable
+backup capacity, and generally be present for service upgrades.
 
 To find processes running different versions than are present on disk, use the
 `xcheckrestart` tool provided by the `xtools` package:
 
-```
-# xbps-install -S xtools
-```
-
 ```
 $ xcheckrestart
 11339 /opt/google/chrome/chrome (deleted) (google-chrome)
 ```
 
 `xcheckrestart` will print out the PID, path to the executable, status of the
-path that was launched (almost always deleted) and the process name.
+path that was launched (almost always `deleted`) and the process name.
 
 `xcheckrestart` can and should be run as an unprivileged user.
 
 ## Kernel Panic After Update
 
-Your system likely ran out of space in `/boot`. XBPS installs kernels and
-requests that hooks such as DKMS and Dracut be run, but it doesn't remove
-kernels that are obsolete. This is left as a task for the administrator to
-permit the retention of obsolete but still booted or known working kernels.
-
-Remove kernels with [vkpurge(8)](https://man.voidlinux.org/vkpurge.8).
+If you get a kernel panic after an update, it is likely your system ran out of
+space in `/boot`. Refer to "[Removing old
+kernels](../config/kernel.md#removing-old-kernels)" for further information.
```

Comment:
If this is the recommended style, I think it should be in the style guide. Usually I don't use quotes when linking to a page using its title, but I think it does look nicer.

Reply:
Mm, i think it depends on the specific context. If literally referring to a section, then yes, it should be in quotes. If the text is not doing that, then it doesn't need to be in quotes. For example:

> For more information, please read the "[Power Management](...)" section.

vs
> Void provides facilities to assist with [power management](...).

If that makes sense to you, i'll update the style guide accordingly.

Reply:
Definitely! Thanks c: